distribution/pom.xml

```diff
@@ -264,6 +287,25 @@
                             </artifactItems>
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>prepare-ballerina-jms-connector</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>unpack</goal>
```

Comment:
Can we reuse the file connector's unpack goal

Reply:
we can rearrange it in a different PR